src/instructlab/model/accelerated_train.py

```diff
@@ -499,7 +499,7 @@ def _training_phase(
 
     if learning_rate is not None:
```

Comment:
I have one other question, isnt `learning_rate` always not None? if so, won't this always happen

Reply:
resolved on slack